views/js/runner/plugins/security/fullScreen.js

```diff
@@ -233,7 +233,9 @@ define([
             }
 
             function enableItem() {
-                testRunner.trigger('enablenav enabletools');
+                if (testRunner.itemRunner !== undefined) {
```

Comment:
please either 
`if (typeof testRunner.itemRunner !== 'undefined)'` or `if (testRunner.itemRunner)` 
for consistency see https://eslint.org/docs/rules/no-undefined